### ISSUE TITLE
VisualStudio does report warnings that are ignored via .editorconfig

### ISSUE THREAD

**Issue by vsfeedback** [dotnet/roslyn#49404]
_This issue has been moved from [a ticket on Developer Community](https://developercommunity2.visualstudio.com/t/VisualStudio-does-report-warnings-that-a/1255530?preview=true)._

---
[severity:It bothers me. A fix would be nice] [regression] [worked-in:16.7.8]
In my project ([TestableHttpClient](https://github.com/dnperfors/TestableHttpClient)) I disabled some warnings specifically for tests by adding the following to the .editorconfig in the folder of the testproject itself:

```
[*.cs]
# CA1707: Identifiers should not contain underscores
dotnet_diagnostic.CA1707.severity = none

# CA2007: Consider calling ConfigureAwait on the awaited task
dotnet_diagnostic.CA2007.severity = silent
```

I also have an .editorconfig in the root of the project.

Since updating VisualStudio to 16.8.0, the editor warns me of both issues again, even though they don't show during compilation.

---
#### Daniel Liuzzi on 11/13/2020, 06:37 AM: 

<p>Happens to us as well. IntelliSense started ignoring .editorconfig altogether after updating to Visual Studio 16.8.<br><br></p>

#### Feedback Bot on 11/16/2020, 09:35 AM: 

<p>This issue is currently being investigated. Our team will get back to you if either more information is needed, a workaround is available, or the issue is resolved.</p>

---
### Original Solutions
(no solutions)

**Comment by mavasani** [comment]
I am unable to repro the issue on 16.8.1 with the repro provided for https://github.com/dnperfors/TestableHttpClient

**Comment by mavasani** [comment]
Closing this as no repro. If you are still able to repro this, please provide concrete repro steps that shows the issue.
